### DISCUSSION TITLE
[2023-03-06] caravel-soc testbench and firmware

### DISCUSSION THREAD

**Discussion by kevinjantw** [bol-edu/caravel-soc#6] [General]
[caravel-soc_testbench&firmwave.pdf](https://github.com/bol-edu/caravel-soc/files/10899276/caravel-soc_testbench.firmwave.pdf)

**Comment by ctc8631** [comment]
May I ask for a pdf version, that because I don't have Microsoft Office, and the layout will be incorrected if I use other program to open it.

**Reply by kevinjantw** [reply]
update
